ios/RCTFBSDK/core/RCTFBSDKSettings.m

```diff
@@ -29,8 +29,8 @@ - (dispatch_queue_t)methodQueue
 
 RCT_EXPORT_METHOD(setAdvertiserTrackingEnabled:(BOOL)ATE resolver:(RCTPromiseResolveBlock)resolve rejector:(RCTPromiseRejectBlock)reject)
 {
-  FBSDKSettings.sharedSettings.advertiserTrackingEnabled = ATE;
-  resolve(@(true)); // true means successfully changed
+  [FBSDKSettings.sharedSettings setAdvertiserTrackingEnabled:ATE];
+  resolve(@(ATE)); // true means successfully changed
```

Comment:
This should stay true? Or the comment should go away, because if we pass `ATE=false`, then the comment is not true.

Reply:
Good catch - this comment needs to be addressed @bonesyblue